doc/man/8/rbd.rst

```diff
@@ -433,15 +455,6 @@ Commands
 :command:`mv` *src-image-spec* *dest-image-spec*
   Rename an image.  Note: rename across pools is not supported.
 
-:command:`nbd ls`
-  Show the list of used nbd devices via the rbd-nbd tool.
-
-:command:`nbd map` [--device *device-path*] [--read-only] *image-spec* | *snap-spec*
-  Map the specified image to a block device via the rbd-nbd tool.
-
-:command:`nbd unmap` *device-path*
-  Unmap the block device that was mapped via the rbd-nbd tool.
-
```

Comment:
Commands `map`, `unmap`, `showmapped` should also be dropped.

Reply:
@dillaman On that note, I don't really understand the need to deprecate `rbd map`, `rbd unmap` and `rbd showmapped`.  Even if `rbd showmapped` is weirdly named, they are short, sweet and to the point ;)  They also have been around forever, so if the new `rbd device` commands are added, I'd like to keep the old ones as synonyms.

Reply:
One of the need to deprecate `rbd showmapped` was discussed in #19704: `rbd showmapped` produces invalid xml, so `rbd device list` produces a different output (see "krbd: add krbd_list and depricate krbd_showmapped function" commit in this PR).

And I like "device" prefix in the command as it groups these commands in one section when `rbd` commands are listed alphabetically. Probably we could keep `rbd map|unmap|showmapped` forever (without a deprecation warning, as aliases, still `showmapped` producing old format output) but to me in long term it might be more confusing to users.

Reply:
Hmm, that's probably not the only place where an XML element name ends up starting with (or being) a number.  I wonder if we should teach `XMLFormatter` to escape these as `_xHHHH_` per https://msdn.microsoft.com/en-us/library/system.xml.xmlconvert.encodename(v=vs.110).aspx or similar.

Reply:
A quick grep just within rbd revealed `do_lock_list`, which opens with a lock cookie, which is a free-form string.  I'm not sure that's enough to deprecate a command in my book.

Reply:
Honestly, the fact that is has been broken for so long probably tells us that no one is using the XML output from ```rbd showmapped```. Encoding the id in hex would be backwards incompatible to any users out there. We've always stated that the output from the CLI is not guarenteed to be stable, so perhaps we just change it, release note it, and move on.